### PR TITLE
Add subscription support for client

### PR DESCRIPTION
**Pull request by edgarmueller**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,99 @@ The following table shows the current WS endpoints:
...
+
+### REST API Example
+
+```Java
+// You can customize the underlying okhttp instance by passing it in as a 1st parameter 
+ModelServerClient client = new ModelServerClient("http://localhost:8081/api/v1/");
+
+// perform simple GET
+client.get("SuperBrewer3000.json")
+      .thenAccept(response -> System.out.println(response.body()))
```
Comment (ndoschek) [inline README.md]: Missing semicolon at line end

File: README.md [hunk 2]
```diff
@@ -57,6 +57,99 @@ The following table shows the current WS endpoints:
...
+
+// perform same GET, but expect XMI format      
+client.get("SuperBrewer3000.json?format=xmi")
+      .thenAccept(response -> System.out.println(response.body()))
```
Comment (ndoschek) [inline README.md]: Missing semicolon at line end

File: com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/common/SessionController.java [hunk 3]
```diff
@@ -69,27 +69,27 @@ public void unsubscribe(WsContext ctx) {
...
 	}
 
-	private void broadcastModelUpdate(String modeluri, @Nullable EObject updatedModel, Context context) {
+	private void broadcastModelUpdate(String modeluri, @Nullable EObject updatedModel) {
 		if (modelUrisToClients.containsKey(modeluri)) {
 			getOpenSessions(modeluri)
 				.forEach(session -> {
 					try {
-						session.send(JsonResponse.data(encoder.encode(context, updatedModel)));
+						session.send(JsonResponse.data(encoder.encode(session, updatedModel)));
```
Comment (ndoschek) [inline com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/common/SessionController.java]: Broadcasting a deletion of a model which has subscribers throws exception: encoder expects EObject, but updateModel is null

Import of Context can be removed as well

**Review by ndoschek — CHANGES_REQUESTED** [review]
Looks good to me! The examples are very well documented and straight forward to use in a Java client.
One small issue came up when deleting a model which has subscribers (please see my comment).

**Comment by edgarmueller** [comment]
@ndoschek Thanks for the review! I've updated the PR

**Review by ndoschek — APPROVED** [review]

**Pull request merged**